SharpPcap/WinDivert/WinDivertHeader.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+namespace SharpPcap.WinDivert
+{
+    public class WinDivertHeader : ICaptureHeader
```

Comment:
`WinDivertCapture` still needed?

Reply:
It isn't after the latest commit to flatten some of the GetNextPacket() methods, let me remove.